packer/install-dependencies.sh

```diff
@@ -63,7 +63,7 @@ echo "  - nvidia-docker2 nvidia-container-runtime" >> ${RELEASE_NOTES_FILEPATH}
 installClearContainersRuntime
 
 VNET_CNI_VERSIONS="
-1.0.22
+1.0.23
```

Comment:
let's make this additive instead (keep the last two supported versions 1.0.22 and 1.0.23) in case we have to revert it allows us to not re-publish a version

Reply:
Done!